src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -30,9 +33,11 @@
 
 
 # TODO(2528577): we should run these test with recording mode.
+@pytest.mark.usefixtures("pf", "runtime")
 @pytest.mark.e2etest
-class TestFlowRun:
-    def test_run_bulk(self, remote_client, pf, runtime):
+class TestFlowRun(PFAzureIntegrationTestCase):
+    @fixture_provider
```

Comment:
why need this provider? Our tests should all be pytest test case?

Reply:
no need to this decorator any more